Formula/h/hckr.rb

```diff
@@ -0,0 +1,23 @@
+class Hckr < Formula
+  include Language::Python::Virtualenv
+
+  desc "hckr - Awesome CLI for developers"
+  homepage "https://pypi.org/project/hckr/"
+  url "https://files.pythonhosted.org/packages/source/h/hckr/hckr-0.2.4.tar.gz"
+  sha256 "21d40e1275e05cf43a6cc0f6f72b15c378c1011fcc4f8577d3658c3a2221fbb9"
+  license "MIT"
+
+  depends_on "python@3.8"
+
+  def install
+    venv = virtualenv_create(libexec, "python3.8")
+    venv.pip_install "pip"
+    venv.pip_install "packaging" # TODO: remove it in new releases
+    system libexec/"bin/pip", "install", "--verbose", "."  # Install package and dependencies
+    bin.install_symlink libexec/"bin/hckr"
+  end
+
+  test do
+    system "#{bin}/hckr", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.